### DISCUSSION TITLE
Menu item "Check for Updates ..."

### DISCUSSION THREAD

**Discussion by fretwurst** [posit-dev/positron#4100] [Feedback] [answered]
Positron is awesome! This definitely solves the issue of having R-Studio and VS-Code running simultaneously. :-) Since Positron is developing so quickly and excitingly, I think a menu item "Check for Updates..." would be great!

**Comment by petetronic** [comment, marked as answer]
@fretwurst thank you for trying out Positron! We plan to add an update mechanism, this issue is tracked via https://github.com/posit-dev/positron/issues/1837
